package.json

```
@@ -39,13 +39,13 @@
   },
   "homepage": "https://github.com/wanadev/changelogify#readme",
   "dependencies": {
-    "commander": "^4.1.0",
-    "inquirer": "^7.0.3",
-    "simple-git": "^3.27.0"
+    "commander": "14.0.0",
+    "inquirer": "12.9.3",
+    "simple-git": "3.28.0"
   },
   "devDependencies": {
-    "eslint": "^7.32.0",
+    "eslint": "9.31.0",
     "eslint-config-airbnb-base": "^14.2.1",
```

Should we also pin to latest version ?

```suggestion
    "eslint-config-airbnb-base": "15.0.0",
```

---

If we keep this package (cf. your comment here https://github.com/wanadev/changelogify/pull/40#issuecomment-3241491545), sure.